### ISSUE TITLE
Feature Request: Custom Catenary 

### ISSUE THREAD

**Issue by CarpetCrumbs** [OpenRCT2/OpenLoco#84]
The default overhead wire for locomotion is a bit boring, and depending on what country you're from, it can look out of place. However a function to be able to  1. change the catenary type, and 2. change how far apart the poles can be. that would be a major improvement for scenery
for instance, maybe some PRR catenary for NEC routes

![image](https://user-images.githubusercontent.com/35979788/35625006-bdcd6d0e-065e-11e8-8b22-b60427d7f374.png)

**Comment by AaronVanGeffen** [comment]
This is a lovely feature request, but a little early considering the state the game is in… (Just managing expectations...)

**Comment by CarpetCrumbs** [comment]
I'm aware of it yes. It's a good idea going forward though.

**Comment by Gymnasiast** [comment]
Like almost everything, these appear to be pluggable objects in Locomotion.


![screenshot4](https://user-images.githubusercontent.com/1478678/35647225-2900b9b4-06d2-11e8-9fc6-cc0ca81e1ac3.png)

**Comment by CarpetCrumbs** [comment]
Yes but you are limited to the amount of track mods you are allowed to use, i believe the max is 2 or 3? so if you were using any type of new catenary and it involved multiple sections it would be incomplete

**Comment by Gymnasiast** [comment]
8, as you can see in the screenshot.

Something that could be addressed in a new save format, but that's something for much, _much_ later.

**Comment by CarpetCrumbs** [comment]
believe it or not in the trackst.xml the limit is not 8. even though the scenario editor says you can select 8.

**Comment by Gymnasiast** [comment]
That doesn't sound like a Locomotion file to me. It's not present in my Locomotion installation, at any rate.

**Comment by duncanspumpkin** [comment]
That's loco tools file name.

**Comment by jgottula** [comment]
On a somewhat related note, something that really bothers me is that the side of the track that the power poles appear on (left vs right) depends on the direction the track piece was actually facing when you built it. For virtually every other practical purpose I can think of, there is zero difference between whether you built a straight track piece in one direction, or in the direction 180 degrees opposite that; it'll appear and function identically in either case. But the power poles are different; they know the secret original direction the track pieces were built in, and always appear on the *left* side of the track relative to the direction you built the segment. (For straight pieces anyway; turns work a little bit differently, I believe.)

This leads to intense OCD pressures in my head, where I drive myself insane fussing over exactly how I build the tracks to make sure that I consistently build my dual-track lines in such a manner that (a) the power poles will not be inconsistent and randomly switch from one side of the track to the other due to being careless and building my track segments in different directions; and (b) so that the poles also end up with some kind of pleasing appearance, e.g. all power poles on the outside side of each of the tracks in a dual-track line, or all on the inside side so that they're all together.

I need help... I think I have a problem... 😐
